client/executor/src/native_executor.rs

```diff
@@ -408,13 +423,15 @@ impl<D: NativeExecutionDispatch> NativeElseWasmExecutor<D> {
 	/// `max_runtime_instances` - The number of runtime instances to keep in memory ready for reuse.
 	///
 	/// `runtime_cache_size` - The capacity of runtime cache.
+	///
+	/// `allow_missing_func_imports` - Ignore missing function imports.
```

Comment:
You rolled back the parameter but you didn't roll back the comment.

Reply:
Ok, fixed.